Makefile

```diff
@@ -228,9 +227,25 @@ add-license: addlicense
 
 # Build the OCI image
 .PHONY: oci-build
-oci-build: generate manifests manager test-envtest
+oci-build: manifests generate fmt vet test-envtest
 	BUILDAH_FORMAT=docker $(IMAGE_BUILDER) build -t $(OPERATOR_IMG) .
 
+# PLATFORMS defines the target platforms for the manager image to provide support to multiple
+# architectures. (i.e. make docker-buildx OPERATOR_IMG=quay.io/cryostat/cryostat-operator:2.3.0).To use this option you need to:
+# - use docker as IMAGE_BUILDER.
+# - able to use docker buildx . More info: https://docs.docker.com/build/buildx/
+# - have enable BuildKit, More info: https://docs.docker.com/develop/develop-images/build_enhancements/
+# To properly provided solutions that supports more than one platform you should use this option.
+PLATFORMS ?= linux/arm64,linux/amd64,linux/s390x,linux/ppc64le
+.PHONY: docker-buildx
+docker-buildx: manifests generate fmt vet test-envtest ## Build OCI image for the manager for cross-platform support
+	# copy existing Dockerfile and insert --platform=${BUILDPLATFORM} into Dockerfile.cross, and preserve the original Dockerfile
+	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{BUILDPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{BUILDPLATFORM\}/' Dockerfile > Dockerfile.cross
+	- docker buildx create --name project-v3-builder
+	docker buildx use project-v3-builder
+	- docker buildx build --platform=$(PLATFORMS) --tag $(OPERATOR_IMG) -f Dockerfile.cross .
+	- docker buildx rm project-v3-builder
+	rm Dockerfile.cross
```

Comment:
So far, I've just tested this portion. I had to make some edits to bring it in line with what's shown here:
https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.25.0/#kubebuilder-bump-brought-changes-that-modified-the-makefile-scaffolding-and-also-multiarch-changes-are-also-included

Basically, we need to add the `--push` argument and also the final bullet point in the comments about needing push access. I wasn't able to make it work locally without pushing for some reason. Here's the resulting image:
https://quay.io/repository/ebaron/cryostat-operator/manifest/sha256:5aaba17aa8cd2deaa80143f9c39cba5e00942f8d17b965baca7ef99c94884039

Once we start publishing multi-arch images, I think it would be good to add a podman/buildah equivalent like this:
https://sdk.operatorframework.io/docs/advanced-topics/multi-arch/
Having the Docker option is good for completeness though.

Reply:
I tried with `-o type=image` without `--push`, I could not find the resulting image/manifest list under `docker images`. I'm not sure why.

Reply:
I found this: https://github.com/docker/roadmap/issues/371. Seems like its not possible yet (available in some beta version tho). I will add `--push` back then.

Reply:
Good find!